swagger-brake/src/main/java/io/redskap/swagger/brake/core/model/Schema.java

```diff
@@ -120,7 +120,7 @@ public Set<String> getRequiredAttributeNames() {
                     StringJoiner stringJoiner = new StringJoiner(".");
                     pieces.subList(0, i + 1).forEach(stringJoiner::add);
                     String attrToSearchFor = stringJoiner.toString();
-                    Boolean isRequired = attributeRequiredMap.get(attrToSearchFor);
+                    boolean isRequired = attributeRequiredMap.get(attrToSearchFor);
```

Comment:
Hmm, I think we need to have it like this: `BooleanUtils.toBoolean(...) `
Otherwise if the required flag is not filled in the Swagger doc, it might just throw an NPE due to unboxing a null value.

Reply:
Sure will update it